static/src/js/containers/SidebarContainer/Facets/Facets.scss

```diff
@@ -42,11 +47,16 @@
   align-self: flex-end;
 }
 
+.facets__selected {
+  padding: 0.2rem 0.4rem 0.2rem 1.1rem;
+  font-size: 0.875rem;
+  color: $color__black--400;
```

Comment:
Can we or do we want to define our colors as variables?

Reply:
Thats what this is :)